src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -637,6 +637,28 @@ def handleCondorStatusService(self):
 
         return
 
+    def handleEnforceGUIDInFileName(self):
+        """
+        _handleEnforceGUIDInFileName_
+
+        Enable enforceGUIDInFileName for CMSSW releases that support it.
+        """
+        # only check this if we are in the first step of a workflow as output files of chained processing
+        # steps will not have correct GUID information
+        if self.step.data._internal_name == "cmsRun1":
+            self.logger.info("Evaluating if release %s supports enforceGUIDInFileName parameter...", self.getCmsswVersion())
+
+            # enable if release supports enforceGUIDInFileName and parameter is not set
+            if isEnforceGUIDInFileNameSupported(self.getCmsswVersion()) and not hasattr(self.process.source, "enforceGUIDInFileName"):
```

Comment:
How about we split this line in two to avoid having it too long? Perhaps the same for the other two logging lines here.
You could also gain 4 spaces by inverting the logic, e.g.:
```
if self.step.data._internal_name != "cmsRun1":
    self.logger.info("Not evaluating enforceGUIDInFileName parameter for step %s",
                                self.step.data._internal_name)
    return
# enable if release supports enforceGUIDInFileName and parameter is not set
            if isEnforceGUIDInFileNameSupported(self.getCmsswVersion()) and  ....
```